Cargo.toml

```diff
@@ -19,13 +20,40 @@ include = [
     "/LICENSE-APACHE",
 ]
 
+[features]
+default = ["nb", "linux-embedded-hal"]
```

Comment:
If requiring `linux-embedded-hal`, bare-metal targets will not build

Reply:
See below.